presidentielles/sondages.json

```diff
@@ -0,0 +1,2094 @@
+{
+    "20210830_harris":
+    {
+        "institut": "Harris interactive",
+        "commanditaires": ["Challenges"],
+        "date_debut": "2021-08-27",
+        "date_fin": "2021-08-30",
+        "date_publication": "2021-09-01",
+        "echantillon": 1083,
+        "echantillon_type": "Inscrits sur les listes électorales",
```

Comment:
Ok, I think I get what this means now. I think this should be renamed `sample_type`, that'd be clearer